core/src/main/scala/cats/syntax/option.scala

```diff
@@ -170,4 +171,28 @@ final class OptionOps[A](val oa: Option[A]) extends AnyVal {
    * }}}
    */
   def orEmpty(implicit A: Monoid[A]): A = oa.getOrElse(A.empty)
+
+  /**
+    * Lift to a F[A] as long as it has an ApplicativeError[F] instance
+    *
+    * Example:
+    * {{{
+    * scala> import cats.implicits._
+    *
+    * scala> Some(1).liftTo[Either[String, ?]]("Empty")
+    * res0: scala.Either[String, Int] = Right(1)
+    *
+    * scala> Option.empty[Int].liftTo[Either[String, ?]]("Empty")
+    * res1: scala.Either[String, Int] = Left(Empty)
+    * }}}
+    */
+  def liftTo[F[_]]: LiftToPartiallyApplied[F, A] = new LiftToPartiallyApplied(oa)
```

Comment:
I'm surprised this works because `LiftToPartiallyApplied` is marked `private[cats]` … I guess I can have a value of this type but I can't declare or ascribe it. Interesting trick.

Reply:
this is the fun of working with scala.